.pre-commit-config.yaml

```diff
@@ -0,0 +1,42 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-yaml
+        exclude: '^deploy/'
```

Comment:
Since we are not handling deployment, I didn't want to check those files @dpoirier. @aldenstpage, does CC use `pre-commit` for running test/checks? Would you find it useful when @TimidRobot  sets up `SALT` to have changes in those follows checked/formatted?

Reply:
Oh right, then we shouldn't have anything under deploy/ so no need to exclude it.

Reply:
@dpoirier, pushed changes to address the bullet points above. I have removed `licenses` app since we'll pull that in from your PR.